input_parser.star

```diff
@@ -89,6 +89,9 @@ DEFAULT_ACCOUNTS = {
     # proofsigner
     "zkevm_l2_proofsigner_address": "0x7569cc70950726784c8D3bB256F48e43259Cb445",
     "zkevm_l2_proofsigner_private_key": "0x77254a70a02223acebf84b6ed8afddff9d3203e31ad219b2bf900f4780cf9b51",
+    # l1testing
+    "l1_deposit_account": "0xC196E8d1D15e15e5703f5Dba9F2b5d6ec35b5bbf",
+    "l1_deposit_account_private_key": "7ffefe81df12e9aeebba4f3671ece167c72aa0fe519a4e66e4c6f7630617b8c3",
```

Comment:
I'm not sure where this account comes from. The accounts are generated currently from the same mnemonic. If you see the comment above, that command automatically generates thie code. You're appending and account that doesn't come from the same mnemonic.

If this account is the default account used by assertoor for the test, then I think you can specifically define a variable in the contract setup script and do something like

```
l1_deposit_account="0xC196E8d1D15e15e5703f5Dba9F2b5d6ec35b5bbf"
fund_account_on_l1 "l1testing" "$l1_deposit_account"
```